src/HeroUtil.cs

```diff
@@ -235,5 +239,18 @@ static byte GetRandomSliderValue(byte v1, byte v2, byte extraRandomization = 4,
             // Commit the new body properties, and we're done!
             SetStaticBodyProperties(hero, heroSliders.GetStaticBodyProperties());
         }
+
+        private delegate TextObject GetHeroNamePropertyDelegate(Hero instance);
+        private static readonly MethodInfo? GetHeroNameProperty = AccessTools.PropertyGetter(typeof(Hero), nameof(Hero.Name));
+        private static readonly AccessTools.FieldRef<Hero, TextObject>? GetHeroNameField = AccessTools.FieldRefAccess<Hero, TextObject>("Hero");

```

Comment:
I think the string that would need to be passed into `AccessTools.FieldRefAccess<,>` is the name of the field (`Name`), not the class as it is here (`Hero`).

https://harmony.pardeike.net/api/HarmonyLib.AccessTools.html#HarmonyLib_AccessTools_FieldRefAccess__2_System_String_